src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -2225,6 +2225,9 @@ struct HeatBalanceData : BaseGlobalStruct
     Array1D<Real64> NominalR;                       // Nominal R value of each material -- used in matching interzone surfaces
     Array1D<Real64> NominalRforNominalUCalculation; // Nominal R values are summed to calculate NominalU values for constructions
     Array1D<Real64> NominalU;                       // Nominal U value for each construction -- used in matching interzone surfaces
+    Array1D<Real64> NominalUBeforeAdjusted;         // Nominal U value for glazing system only
+    Array1D<Real64> CoeffAdjRatioIn;                // Conductive and radiative coefficient adjustment ratio
+    Array1D<Real64> CoeffAdjRatioOut;               // Conductive and radiative coefficient adjustment ratio
```

Comment:
The comments here are miss-informative, the are not "Conductive and radiative."  It should be made clear that it is _Convection_ coefficient adjustment ratio for modeling increase wetted area ratio.

Reply:
Indeed, I will change the comments here to "Convection coefficient adjustment ratio for modeling increased wetted area ratio of inside(outside) surfaces"

Reply:
Modified.